footprints/templates/main/index.html

```diff
@@ -85,8 +85,12 @@
                           <li>an extant copy of an imprint</li>
                             <li>a copy owned in the past whose whereabouts are unknown</li>
                       </ul>
-                      <input type="text" class="home-search-text"></input>
-                      <input type="submit" class="home-search-submit" value="Search"></input>
+                      
+                      <form method="get" action="{% url 'search' %}">
+                          <input type="text" name='q' class="home-search-text"></input>
+                          <input type="hidden" name="models" value="main.footprint" />                          
+                          <input type="submit" class="home-search-submit" value="Search"></input>
```

Comment:
I don't think this `<input>` or the one on line 90 should have a trailing `</input>`. Instead, change to `<input />` like on line 91.